Documentation/SpatialAwareness/SpatialObjectMeshObserver.md

```diff
@@ -1,78 +1,78 @@
+A convenient way to provide environment mesh data in the Unity editor is to use the [SpatialObjectMeshObserver](xref:) class. The *Spatial Object Mesh Observer* is an editor-only data provider for the [Spatial Awareness system](SpatialAwarenessGettingStarted.md) that enables importing 3D model data to represent a spatial mesh. One common use of the *Spatial Object Mesh Observer* is to import data scanned via a Microsoft HoloLens to test how an
 experience adapts to different environments from within Unity.
 
 ## Getting started
 
-To enable the spatial object mesh observer, you will need to configure the spatial awareness system.
+This guide will walk through setting up a *Spatial Object Mesh Observer*. There are three key steps to enable this feature.
 
-1. Add a new spatial observer
+1. Add a *Spatial Object Mesh Observer* to the Spatial Awareness system profile
+1. Set the Environment Mesh Data object
+1. [Configure rest of the Mesh Observer profile properties](ConfiguringSpatialAwarenessMeshObserver.md)
 
-![Add Spatial Observer](../../Documentation/Images/SpatialAwareness/AddObserver.png)
+### Set up a *Spatial Object Mesh Observer* profile
 
-2. Select the SpatialObjectMeshObserver 
+1. Select the desired *Mixed Reality Toolkit* configuration profile or select the *Mixed Reality Toolkit* object in scene
+1. Open or expand the *Spatial Awareness System* tab
+1. Click on *"Add Spatial Observer"* button
 
-![Select Spatial Object Mesh Observer](../../Documentation/Images/SpatialAwareness/SelectObjectObserver.png)
+    ![Add Spatial Observer](../../Documentation/Images/SpatialAwareness/AddObserver.png)
 
-3. Select the desired Spatial Mesh Object  
+1. Select the *SpatialObjectMeshObserver* type
 
-![Select the Mesh Object](../../Documentation/Images/SpatialAwareness/ObjectObserverProfile.png)
+    ![Select Spatial Object Mesh Observer](../../Documentation/Images/SpatialAwareness/SelectObjectObserver.png)
 
-By default, the observer is configured with an example model. This model was created using a Microsoft HoloLens.
+1. Select the desired *Spatial Mesh Object*. By default, the observer is configured with an example model. This model was created using a Microsoft HoloLens but it is possible to [create a new scan mesh object](#acquiring-environment-scans).
+1. [Configure rest of the Mesh Observer profile properties](ConfiguringSpatialAwarenessMeshObserver.md)
 
-## Acquiring environment scans
+    ![Select the Mesh Object](../../Documentation/Images/SpatialAwareness/ObjectObserverProfile.png)
 
-### Windows Device Portal
+### Spatial Object Mesh Observer profile notes
 
-The [Windows Device Portal](https://docs.microsoft.com/en-us/windows/mixed-reality/using-the-windows-device-portal) can be used to download the spatial mesh, as a .obj file, from a Microsoft HoloLens device.
+Since the *Spatial Object Mesh Observer* loads data from a 3D model, it does not honor some of the standard mesh
```

Comment:
Random question: Why wouldn't I just add the obj I downloaded directly to my scene? What would be the disadvantage of that? Why would i want to use a spatial mesh observer?

Reply:
@davidkline-ms ?

Reply:
> Why wouldn't I just add the obj I downloaded directly to my scene?

Adding the obj to the scene would not allow the developer to exercise the code that would run when the application is using live spatial data.